styles/globals.css

```diff
@@ -103,4 +103,11 @@
   font-size: 13px;
   line-height: 100%;
   letter-spacing: 1px;
-}
\ No newline at end of file
+}
+
+/*container for the whole page*/
+.container {
```

Comment:
If this container is the one for the whole page, why not change its name to something else?